document/js-api/index.bs

```diff
@@ -729,9 +729,20 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
 
 <h3 id="globals">Globals</h3>
 
+<pre class="idl">
+enum WebAssemblyValueType {
+  "i32",
+  "i64",
+  "f32",
+  "f64"
+};
+</pre>
+
+Note: the WebAssemblyValueType enum is subject to future changes; several incomming WebAssembly GC proposals will add new values or extend the types. 
```

Comment:
Nit: it's not just the GC proposal. In general, I wouldn't mention proposals in the spec. How about:

"Note: This type may be extended with additional cases in future versions of WebAssembly. Client code should not assume that no other constants can occur."